workers/scanner/src/main/resources/application.conf

```diff
@@ -48,3 +48,6 @@ scanner {
     password = ${?SCANNER_RECEIVER_TRANSPORT_PASSWORD}
   }
 }
+
+gitConfigUrlInsteadOf = "https://github.com=ssh://git@github.com,https://github.com/=git@github.com:"
```

Comment:
I would rather use an empty default. The URLs listed here could be perfectly fine in some environment.